libkernelflinger/storage.c

```diff
@@ -86,10 +86,10 @@ static EFI_STATUS identify_storage(EFI_DEVICE_PATH *device_path,
 		, &STORAGE(STORAGE_SATA)
```

Comment:
Why do you put the comma at the beginning of the line.  This is violation of the coding style !